packages/create-cloudflare/src/frameworks/angular/index.ts

```diff
@@ -31,26 +31,25 @@ const configure = async (ctx: PagesGeneratorContext) => {
 	await runCommand(`${npx} ng analytics disable`, {
 		silent: true,
 	});
-	await addSSRAdapter();
-	await installCFWorker(ctx);
-	await updateAppCode();
-	updateAngularJson(ctx);
+
+  await installCFWorker(ctx);
+  await updateAppCode();
+  updateAngularJson(ctx);
 };
 
 const config: FrameworkConfig = {
 	generate,
 	configure,
 	displayName: "Angular",
 	getPackageScripts: async () => ({
-		process:
-			"node ./tools/copy-worker-files.mjs && node ./tools/copy-client-files.mjs && node ./tools/bundle.mjs",
-		"pages:build": `${npm} run build:ssr && ${npm} run process`,
-		start: `${npm} run pages:build && wrangler pages dev dist/cloudflare ${await compatDateFlag()} --experimental-local`,
-		deploy: `${npm} run pages:build && wrangler pages deploy dist/cloudflare`,
+    start: `${npm} run pages:build && wrangler pages dev dist/cloudflare ${await compatDateFlag()} --experimental-local`,
+    process: "node ./tools/copy-files.mjs && node ./tools/alter-polyfills.mjs",
+    "pages:build": `ng build && ${npm} run process`,
+    deploy: `${npm} run pages:build && wrangler pages deploy dist/cloudflare`,
 	}),
 	deployCommand: "deploy",
 	devCommand: "start",
-	testFlags: ["--routing", "--style", "sass"],
+	testFlags: ["--ssr", "--style", "sass"],
```

Comment:
Not sure what these testFlags are used for.

Reply:
These are used by the e2e tests to bypass any interactive prompts in `@angular/cli` by specifying any required input via command line props. Anything in this array will be appended to the `npx @angular/cli new` call during e2e tests.

Reply:
Thanks for the clarification.